src/main/java/bio/terra/service/dataset/flight/ingest/IngestMapKeys.java

```diff
@@ -3,6 +3,14 @@
 public final class IngestMapKeys {
   private IngestMapKeys() {}
 
-  public static final String INGEST_STATISTICS = "ingestStatistics";
-  public static final String STAGING_TABLE_NAME = "stagingTableName";
+  public static final String PREFIX = "ingest-";
```

Comment:
Thanks for doing this, although now I wonder if it will be a problem for any flights that are running across system updates. Once we write data to a flightmap using a key, can we change the key without breaking flights that are running?

Reply:
That's a great point - I think it could break a flight if the key changes between the time we write it to the flight map and try to pull it back out of the flight map. 

I think I'll just undo the changes to the INGEST_STATISTICS and STAGING_TABLE_NAME keys.

Reply:
Is there a reason why we want this prefix? They're all already grouped in the IngestMapKeys.

Reply:
After discussing with Trevyn, we're thinking that we could keep the prefix here, even for the existing keys. We will just have to shepherd through these changes and communicate with users who might be performing an ingest while the data-repo version is upgraded.